src/chapters/data-and-variables/more-on-variables.rst

```diff
@@ -0,0 +1,196 @@
+More On Variables
+=================
+
+The previous section covered creating, evaluating, and reassigning variables. This section will cover some additional, more nuanced topics related to variables.
+
+Creating Constants With ``const``
+---------------------------------
+
+One the key features of variables that we have discussed so far is their ability to change value. We can create a variable with one value, and then reassign it to another value.
+
+.. sourcecode:: js
+
+   let programmingLanguage = "JavaScript";
+   programmingLanguage = "Python";
+
+In some situations, however, we will want to create variables that *should not* change value. Many programming languages, including JavaScript, provide a mechanism for creating variables that may only have a single, fixed value.
```

Comment:
I recommend rewording this.

Reply:
@gildedgardenia Can you say a bit more about which part is confusing? Maybe a suggested way to reword it? I'm not clear on what needs to be improved on this one.

Reply:
It can just be shortened: "In some situations, we want to create variables that cannot change value. Many programming languages, including Javascript, provide mechanisms for programmers to make variables that are constant."